docs/01-app/01-getting-started/07-server-and-client-components.mdx

```diff
@@ -0,0 +1,494 @@
+---
+title: How to use Server and Client Components
+nav_title: Server and Client Components
+description: Learn how you can use React Server and Client Components to render parts of your application on the server or the client.
+related:
+  title: Next Steps
+  description: Learn more about the APIs mentioned in this page.
+  links:
+    - app/api-reference/directives/use-client
+    - app/api-reference/file-conventions/route
+---
+
+By default, Next.js uses React [Server Components](https://react.dev/reference/rsc/server-components) which let you render parts of your UI on the server, optionally cache the result, and stream it to the client. When needed, you can use [Client Components](https://react.dev/reference/rsc/use-client) to add interactivity and use browser APIs.
+
+This page will go through how Server and Client Components work, and common examples of when to use them.
+
+## How do Server and Client Components work?
+
+### On the server
+
+On the server, Next.js uses React's APIs to orchestrate rendering. The rendering work is split into chunks, by individual route segments ([layouts and pages](/docs/app/getting-started/layouts-and-pages)):
+
+- **Server Components** are rendered into a special data format called the React Server Component Payload (RSC Payload).
+- **Client Components** are rendered into HTML.
```

Comment:
I find this distinction confusing. I think it's more correct to explain that they're all rendered into RSC payload, and that RSC payload itself can be rendered to HTML (but also contains JSON props).